kyo-core/shared/src/test/scala/kyo/KyoAppTest.scala

```diff
@@ -30,6 +34,17 @@ class KyoAppTest extends Test:
             runs <- ref.get
         yield assert(runs == 3)
     }
+    "multiple runs on JS" taggedAs jsOnly in {
```

Comment:
This test is necessary for JVM and JS. We can delete the other one.

```suggestion
    "ordered runs" in {
```